plugins/modules/ec2_vpc_route_table.py

```diff
@@ -47,11 +47,19 @@
     - Required when I(lookup=id).
     type: str
   routes:
-    description: List of routes in the route table.
-        Routes are specified as dicts containing the keys 'dest' and one of 'gateway_id',
-        'instance_id', 'network_interface_id', or 'vpc_peering_connection_id'.
-        If 'gateway_id' is specified, you can refer to the VPC's IGW by using the value 'igw'.
-        Routes are required for present states.
+    description:
+        - >
+          List of routes in the route table.
+        - >
+          Routes are specified as dicts containing the keys 'dest' and one of 'gateway_id',
+          'instance_id', 'network_interface_id', or 'vpc_peering_connection_id'.
+        - >
+          The value of 'dest' is used for the destination match. It may be a IPv4 CIDR block
+          or a IPv6 CIDR block.
+        - >
+          If 'gateway_id' is specified, you can refer to the VPC's IGW by using the value 'igw'.
+        - >
+          Routes are required for present states.
```

Comment:
```suggestion
    description:
        - List of routes in the route table.
        - Routes are specified as dicts containing the keys C(dest) and one of C(gateway_id),
          C(instance_id), C(network_interface_id), or C(vpc_peering_connection_id).
        - The value of C(dest) is used for the destination match. It may be a IPv4 CIDR block
          or a IPv6 CIDR block.
        - If I(gateway_id) is specified, you can refer to the VPC's IGW by using the value C(igw).
        - Routes are required for present states.
```

`C(...)` is used for quoting values, `I(...)` is used for quoting the names of module options.

The `>` YAML block indicators aren't really needed here.

Even better would be to define the suboptions here (see for example [elb_classic_lb](https://github.com/ansible-collections/amazon.aws/blob/main/plugins/modules/elb_classic_lb.py#L34) however you then need to also define that in the [arg_spec](https://github.com/ansible-collections/amazon.aws/blob/main/plugins/modules/elb_classic_lb.py#L2096)